M2/Macaulay2/packages/ResolutionsOfStanleyReisnerRings.m2

```diff
@@ -0,0 +1,592 @@
+    Description
+        Text
+            {\em ResolutionsOfStanleyReisnerRings} is a package for computing certain systems of parameters and finding the Betti numbers of the resolution of the Stanley-Reisner ring of a simplicial complex over certain parameter rings.
+
+            Given a simplicial complex  {\tt D}, there is a {\em universal system of parameters} (USOP) [AR,GS,HM,S] where each parameter {\tt i}  is a sum of the faces of {\tt D} of dimension {\tt i+1}. The {\em colorful system of parameters} (KSOP) is obtained by giving a proper d-vertex coloring to a balanced simplicial complex {\tt B} of dimension {\tt d-1},  and then summing over the colors so that each parameter {\tt j}  is the sum of vertices of color {\tt j}. This package includes routines for computing both USOP and KSOP for simplicial complexes and for computing the graded Betti numbers of the resolutions of the Stanley-Reisner rings over the graded parameter rings.
+
+            References:
+
+            [AR] A. Adams and V. Reiner, A Colorful Hochster Formula and Universal Parameters for Face Rings. Preprint, 2020; arXiv:2007.13021.
+
+            [GS] A.M. Garsia and D. Stanton, Group actions of Stanley-Reisner rings and invariants of permutation groups. Adv. in Math. 51 (1984), 107–201.
+
+            [HM] J. Herzog and S. Moradi, Systems of parameters and the Cohen--Macaulay property. Preprint, 2020; arXiv:2006.16549.
+
+            [S]  D.E. Smith, On the Cohen-Macaulay property in commutative algebra and simplicial topology. Pac. J. Math. 141 (1990), 165–196.
+
+            The goal of this work was primarily to help compute examples to provide evidence for Conjecture 6.1 [AR]. I have tried to generalize most of the functionality to make it useful in other areas.  This is work in progress and many interesting pieces are still missing. All suggestions and contributions are welcome.
+///
+
+      --UL
+    	    --TO "kSOP", " -- computes the colorful system of parameters for the barycentric subdivision of a simplicial complex",
+					--TO "uSOP", " -- computes the universal system of parameters for a simplicial complex",
+
+doc ///
+    Key 
+        barycentricSubdivision 
+    Headline
+        the barycentric subdivision of a simplicial complex. 
+    Usage 
+         barycentricSubdivision D
+         barycentricSubdivision F
+    Inputs
+        D:SimplicialComplex
+        F:List
+            a list of facets defining a simplicial complex
+    Outputs
+        B:SimplicialComplex
+            the barycentric subdivision of a simplicial complex of {\tt D}
+    Description
+        Text
+            The following example uses the simplicial complex on four vertices composed of two diconnected edges.
+        Example
+            S = QQ[a,b,c,d];
+            F = {a*b,c*d};
+            D = simplicialComplex F;
+            barycentricSubdivision D;            
+            barycentricSubdivision F;            
```

Comment:
There's no point in an example if no answers are displayed.  The answers are tiny, so why not display them?